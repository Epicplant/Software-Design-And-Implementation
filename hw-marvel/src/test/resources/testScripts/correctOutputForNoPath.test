# Tests that when a path doesn't exist that the correct output is displayed

# Loads a new graph from a csv file
LoadGraph heroes someUnconnectedHeroes.csv

# Finds shortest path between heroes
FindPath heroes A C