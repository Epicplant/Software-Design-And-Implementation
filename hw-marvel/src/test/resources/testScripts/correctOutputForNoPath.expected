# Tests that when a path doesn't exist that the correct output is displayed

# Loads a new graph from a csv file
loaded graph heroes

# Finds shortest path between heroes
path from A to C:
no path found